UI/index.html

```diff
@@ -24,13 +25,12 @@
         </a>
         <div class="navbar-links">
             <ul>
-                <div> <input type="text" placeholder="Search .." id="search">
+                <div id="searchDiv"> <input type="text" placeholder="Search .." id="search">
```

Comment:
Why are you using an ``id`` here instead of a class ?

Reply:
I used id because I was going to use that id only in the search div not anywhere else in that HTML file. It can be changed to class